src/main/conscript/g8/launchconfig

```diff
@@ -2,7 +2,7 @@
   version: 0.7.2
   org: org.foundweekends.giter8
   name: giter8
-  class: giter8.Giter8
+  class: giter8.Giter8App
```

Comment:
Changing entry class names might have unintended consequences (for example launchconfig files that's been installed by conscript refers to Giter8), so I think we should keep this the same if possible.